src/widgets/allergies/allergy-form.component.tsx

```diff
@@ -13,6 +13,7 @@ import {
 import { createErrorHandler } from "@openmrs/esm-error-handling";
 import { useCurrentPatient } from "@openmrs/esm-api";
 import dayjs from "dayjs";
+import { WorkspaceItemProps } from "../../workspace/workspace.resource";
```

Comment:
Will it be necessary for every component that wants to be placed in the workspace to import WorkspaceItemProps? It would be better if the component was not required to "know" about the workspace.